src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -1782,10 +1782,10 @@
   <data name="NavigationToolbarClosePane.Label" xml:space="preserve">
     <value>Close pane</value>
   </data>
-  <data name="NavigationToolbarEnterCompactOverlay.Label" xml:space="preserve">
```

Comment:
Can you rename this in the other files so that we don't lose the translations?

Reply:
We use that only once. I'll check that again

Reply:
I meant that other translation files, otherwise it needs to be retranslated.

Reply:
Do I need to remove strings from all translations too?

Reply:
No